src/main/java/edu/greenblitz/robotName/subsystems/Arm/EndEffector/Roller/SimulationRoller.java

```diff
@@ -0,0 +1,34 @@
+package edu.greenblitz.robotName.subsystems.Arm.EndEffector.Roller;
+
+import edu.greenblitz.robotName.RobotConstants;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.FlywheelSim;
+
+public class SimulationRoller implements IRoller{
+	
+	private FlywheelSim motorSimulation;
```

Comment:
if we are simulating the roller consider using the DCMotorSim class for the simulation and not the flywheel